docs/exts/eliot_metrics.py

```diff
@@ -24,13 +26,13 @@ def build_table(table):
     yield "  ".join("=" * width for width in col_size)
     yield "  ".join(
         header + (" " * (width - len(header)))
-        for header, width in zip(table[0], col_size)
+        for header, width in zip(table[0], col_size, strict=True)
```

Comment:
`zip` changed in Python 3.10 growing a `strict` argument which ensures that it returns full tuples rather than partial tuples.